src/argilla/client/users.py

```diff
@@ -115,16 +114,21 @@ def __str__(self) -> str:
         )
 
     @staticmethod
-    def __active_client(httpx: bool = True) -> Union["httpx.Client", "AuthenticatedClient"]:
+    def __active_client() -> Union["httpx.Client", "AuthenticatedClient"]:
         """Returns the active Argilla `httpx.Client` instance."""
         try:
-            client = active_client().http_client
-            if httpx:
-                client = client.httpx
-            return client
+            return active_client().http_client.httpx
         except Exception as e:
             raise RuntimeError(f"The `rg.active_client()` is not available or not respoding.") from e
 
+    @property
+    def workspaces(self) -> List[str]:
+        try:
+            user = users_api.whoami(self.__client).parsed
```

Comment:
What if the user we want to list the workspaces is not the active one, because we're the owner and we want to list the workspaces for a given user?

We shouldn't use `whoami` here, but a request to the API to get the workspaces for the current user based on an ID or name match, but not using `whoami`, as the activated user may be owner or admin

Reply:
Totally true. I will revert all changes related to the `workspaces` attribute and we'll plan a PR to improve this (maybe we should create a new endpoint to fetch info for an specific user)